docs/readthedocs/models/load_model.md

```diff
@@ -0,0 +1,153 @@
+(load_model)=
+## Load Model Configuration and Load Profiles
+
+SIMONA includes a polymorph approach for load model configuration. Based on parameterization the load model can be a profile model or a random model.
+
+### Load Model
+
+#### General Information
+
+SIMONA supports different load model behaviours, which need some parameterization made by this config entry. It can be found in the config tree at ```simona.runtime.participant.load```.
+
+This sub configuration is two-fold:
+
+Default Configuration
+This is one configuration, that applies to all load models, except of those, that have an individual config assigned. To simplify code, no distinct model class is introduced for the single config. In order to highlight the fact, that this config is a default config, you may write ```uuids = ["default"]```. If you put anything else than this, you will get a warning, that those references are neglected, although the rest will work just fine.
```

Comment:
*This applies to all load models, except for those, which already have an individual config assigned.